db/views/patient_states_over_time_v01.sql

```diff
@@ -0,0 +1,79 @@
+SELECT
+    DISTINCT ON (p.id, month_date)
+    p.id,
+    p.recorded_at,
+    p.deleted_at,
+    p.status,
```

Comment:
Consider adding age, and gender here.

Reply:
Most definitely. They're already stated as potential dimensions for disaggregation in our Questions doc.